socialhome/frontend/stores/streamStore.js

```diff
@@ -116,14 +116,14 @@ function newRestAPI(options) {
         })
         .get({
             action: streamStoreOperations.getProfileAll,
-            path: ({id, lastId = undefined}) => `${Urls["api-streams:profile-all"]({id})}${getLastIdParam(lastId)}`,
+            path: ({uuid, lastId = undefined}) => `${Urls["api-streams:profile-all"]({uuid})}${getLastIdParam(lastId)}`,
```

Comment:
Was trying to find where you change these being passed in but it seems that happens already in the previous PR ;)

Reply:
Yeah, I accidentally got ahead of myself I guess... And that actually broke fetching content from profiles, but it's fixed with this PR.

Reply:
Well I approved it so... :) For master branch it's "move fast break things".